src/math/p5.Vector.js

```diff
@@ -1743,83 +1980,96 @@ p5.Vector = class {
   }
 
   /**
- * Performs spherical linear interpolation with the other vector
- * and returns the resulting vector.
- * This works in both 3D and 2D. As for 2D, the result of slerping
- * between 2D vectors is always a 2D vector.
+ * Calculates a new heading and magnitude that are between two vectors. The
+ * `amt` parameter is the amount to interpolate between the old vector and
+ * the new vector. 0.0 keeps the heading and magnitude equal to the old
+ * vector's, 0.5 sets them halfway between, and 1.0 sets the heading and
```

Comment:
Maybe we can mention specifically that this magnitude property is not present in `lerp` so one knows when to use one over the other

Reply:
Good call, let me know how this draft reads.